### ISSUE TITLE
64 bit Aion bug, Aion can't start on Windows 11 24H2 and later. 

### ISSUE THREAD

**Issue by flroexus** [zzsort/Aion-Version-Dll#20]
64 bit Aion bug, Aion can't start on Windows 11 24H2 and later.

**Comment by Marisa-Chan** [comment]
Try this one https://github.com/Marisa-Chan/Aion-Version-Dll  with new cmdline key `-fix-stack-win11`

(build it yourself ;) )

**Comment by benas80** [comment]
Windows 10 22H2 , still Aion 5.8 can't start (just logo and nothing)

**Comment by Marisa-Chan** [comment]
> Windows 10 22H2 , still Aion 5.8 can't start (just logo and nothing)

What about number of processor cores? If it's 32 and more it's needs affinity fix.
But in any case, where I can get this version to test it by myself?

**Comment by benas80** [comment]
I have 13900k proc., how to do affinity fix ?

**Comment by Marisa-Chan** [comment]
> I have 13900k proc., how to do affinity fix ?

try create "bat" file with:
`start /AFFINITY 7fffffff bin64/aion.bin .....`

to limit use threads to 31(if vcpu >= 32 it's must be limited by -1 thread, because of use signed variable in code which inspects all vcpu's. But generally it's cause crash, so that's why I ask about your Aion binaries version)

**Comment by Marisa-Chan** [comment]
> I have 13900k proc., how to do affinity fix ?

Did it help?

**Comment by benas80** [comment]
Yes, it works, thak You very much,  but i dont how it works on W11, i dont have it installed.8 maj 2025 06:19 Marisa-Chan ***@***.***> napisał(a):
Marisa-Chan left a comment (zzsort/Aion-Version-Dll#20)

I have 13900k proc., how to do affinity fix ?

Did it help?

**Comment by Marisa-Chan** [comment]
@benas80 very good. I think it will also work on w11 too. I plan to try add affinity fix into version.dll. When I make it I will write about it here.

**Comment by benas80** [comment]
I there's an issue when using windowed mode instead of fullscreen — right-clicking to look around causes the camera to jump to an incorrect position every time.Can you check this ?8 maj 2025 12:37 Marisa-Chan ***@***.***> napisał(a):
Marisa-Chan left a comment (zzsort/Aion-Version-Dll#20)
@benas80 very good. I think it will also work on w11 too. I plan to try add affinity fix into version.dll. When I make it I will write about it here.

**Comment by Marisa-Chan** [comment]
`-windowed-mouse-fix` does not solve it?

**Comment by benas80** [comment]
I have this line in cmd line but in window mode is not good. It works normal in fullscreen, but if you click right button + first mouse move = camera jump. Check this please.8 maj 2025 12:51 Marisa-Chan ***@***.***> napisał(a):
Marisa-Chan left a comment (zzsort/Aion-Version-Dll#20)
-windowed-mouse-fix does not solve it?

**Comment by Marisa-Chan** [comment]
Did you try do not add mouse fix at all or do not add windowed fix?

**Comment by benas80** [comment]
Tested with mouse fix added8 maj 2025 13:25 Marisa-Chan ***@***.***> napisał(a):
Marisa-Chan left a comment (zzsort/Aion-Version-Dll#20)
Did you try do not add mouse fix at all or do not add windowed fix?

**Comment by benas80** [comment]
Hmm it's strange, it's working fine with: -win10-mouse-fix AND -windowed-mouse-fix DISABLED :) Windowed and Fullscreen mode

**Comment by Marisa-Chan** [comment]
> Hmm it's strange, it's working fine with: -win10-mouse-fix AND -windowed-mouse-fix DISABLED :) Windowed and Fullscreen mode

Yeah, I'm talk about of this when say to try with multiple combinations.

In any case you may try this https://github.com/Marisa-Chan/Aion-Version-Dll/releases/tag/win10_11_64bit_1.02 and now without of adding any `/affinity mask`.

**Comment by Marisa-Chan** [comment]
@benas80 was there time to test new version?

**Comment by daraeowl** [comment]
> [@benas80](https://github.com/benas80) was there time to test new version?

I've tested it and works really well, except sometimes it still randomly moves the camera to the bottom 
 win 11 latest ver
<img width="1594" height="1183" alt="Image" src="https://github.com/user-attachments/assets/73a7ea1f-7b9c-45f7-aeac-c161664cc8c1" /> or top

**Comment by jeanmarcos-dev** [comment]
Hi @Marisa-Chan i try with -fix-stack-win11, but game still crashed
Is there a workaround to get the game running in 64-bit?

**Comment by Marisa-Chan** [comment]
> Hi [@Marisa-Chan](https://github.com/Marisa-Chan) i try with -fix-stack-win11, but game still crashed Is there a workaround to get the game running in 64-bit?

What client version?(you may share bin64 folder) And did you absolutely sure using version.dll from [my repo](https://github.com/Marisa-Chan/Aion-Version-Dll/releases) ?

**Comment by jeanmarcos-dev** [comment]
@Marisa-Chan Thank you very much for the prompt response. I just compiled the DLL directly from your repository, and everything is working perfectly. Thank you again.

**Comment by blood-creator** [comment]
@Marisa-Chan 
we got the same problem on the 25H2 Win11....will that fix that also witht that file?

**Comment by Marisa-Chan** [comment]
@blood-creator You just can try ;) If it works or not please write here.  It must work if MS do not touch any other winapi functions used by Themida protector

**Comment by blood-creator** [comment]
@Marisa-Chan changed many times those version.dll which u postet here or in your repo,..didnt work, it shows ugly 32 bit xd

**Comment by Marisa-Chan** [comment]
@blood-creator  are you sure?

<img width="1028" height="625" alt="Image" src="https://github.com/user-attachments/assets/5b675bea-bde3-4b91-8bba-2bd2def6eb60" />

**Comment by blood-creator** [comment]
@Marisa-Chan 

hmm weird, maybe the admin can change that ",exe" or Launcher to 64bit ? 
coz mine started always at 32bit in Windows 11 25H2 the newest one...

and not only me, another guys on same server aswell

**Comment by Marisa-Chan** [comment]
@blood-creator  all questions to your server admins and their launcher (suppose ActiveAnticheat crap), because it's works for self-hosted servers.
